src/router.js

```diff
@@ -65,6 +66,7 @@ const Routes = () => (
       <Route path="/verify/:token" component={VerificationSentPage} />
       <Route path="/reset_password/:token" component={CreateNewPassword} />
       <Route path="/team" component={TeamPage} />
+      <Route path="/database" component={DBSettingsPage} />
```

Comment:
Is this the best route you can put the settings page? just inquiring

Reply:
@Mubangizi given that the endpoint hasn't been made and all that is dummy data I figured that all would change when plugging in.

Reply:
ideally it should under `/databases/:dbID/[maybe settings]`